clients/privacy-center/cypress/e2e/consent-i18n.cy.ts

```diff
@@ -1509,7 +1509,149 @@ describe("Consent i18n", () => {
+    const VERIFICATION_CODE = "112358";
+
+    beforeEach(() => {
+      cy.clearAllCookies();
+
+      // Seed local storage with verification data
+      cy.window().then((win) => {
+        win.localStorage.setItem(
+          "consentRequestId",
+          JSON.stringify("consent-request-id")
+        );
+        win.localStorage.setItem(
+          "verificationCode",
+          JSON.stringify(VERIFICATION_CODE)
+        );
+      });
+
+      // Enable GPC
+      cy.on("window:before:load", (win) => {
+        // eslint-disable-next-line no-param-reassign
+        win.navigator.globalPrivacyControl = true;
+      });
+
+      // Intercept sending identity data to the backend to access /consent page
+      cy.intercept(
+        "POST",
+        `${API_URL}/consent-request/consent-request-id/verify`,
+        { fixture: "consent/verify" }
+      ).as("postConsentRequestVerify");
+
+      // Location intercept
+      cy.intercept("GET", GEOLOCATION_API_URL, {
+        fixture: "consent/geolocation.json",
+      }).as("getGeolocation");
+
+      // Experience intercept
+      cy.intercept("GET", `${API_URL}/privacy-experience/*`, {
+        fixture: "consent/experience_privacy_center.json",
+      }).as("getExperience");
+
+      cy.visitWithLanguage("/consent", SPANISH_LOCALE);
+      cy.overrideSettings({ IS_OVERLAY_ENABLED: true });
+    });
+
+    it("displays localized text from experience", () => {
+      cy.getByTestId("consent-heading").contains(SPANISH_MODAL.title);
+      cy.getByTestId("consent-description").contains(SPANISH_MODAL.description);
+    });
+
+    it("displays localized text for gpc banner", () => {
+      cy.getByTestId("gpc.banner.title").contains(SPANISH_MODAL.gpc_title);
+      cy.getByTestId("gpc.banner.description").contains(
+        SPANISH_MODAL.gpc_description
+      );
+    });
+
+    it("displays localized save button", () => {
+      cy.getByTestId("save-btn").contains(SPANISH_MODAL.save_button_label);
+    });
+
+    it("displays localized privacy policy", () => {
+      cy.getByTestId("privacypolicy.link").contains(
+        SPANISH_MODAL.privacy_policy_link_label!
+      );
+      cy.getByTestId("privacypolicy.link")
+        .should("have.attr", "href")
+        .and("include", SPANISH_MODAL.privacy_policy_url!);
+    });
+
+    it("displays localized notice texts", () => {
+      cy.getByTestId("consent-item-pri_notice-analytics-000").contains(
+        SPANISH_NOTICES[1].title
+      );
+      cy.getByTestId("consent-item-pri_notice-analytics-000").contains(
+        SPANISH_NOTICES[1].description
+      );
+    });
+
+    /* Commenting tests for now. The problem is that the test runs with config based consent,
+      which don't support translations. Then, we override with cy.overrideSettings({ IS_OVERLAY_ENABLED: true });
+      to use notices-based consent to run the test, but notices-server has already ran with the previous config
+      */
```

Comment:
May need a second opinion on this, from @NevilleS or @eastandwestwind maybe?  What would be required to make these tests work?

Reply:
Yeah, I generally don't like that the override is applied after load. I think we should implement it in a way of having that setup from the beginning. But that sounds to me like it could be it's own ticket, as to not keep making this PR bigger.
cc @NevilleS

Reply:
Yeah, I understand and empathize - in fact, I'm quite happy to see that you managed to get the _regular_ UI loading under test! I'll ask a few more questions in Slack about this...

Reply:
👍 I'm OK with disabling these tests. That said, personally I like _skipped_ tests more, so can you do something like:
```
// DEFER (PROD-1748): Disable E2E tests for notices served because {xyz}...
describe.skip("when reporting notices served and preferences saved", () => {
...(all these commented out tests here)
});